.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig is awesome: https://EditorConfig.org
+# If you are using VS Code consider using the following extension
+# https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{rst,js,py}]
+charset = utf-8
+
+[*.{rst,md}]
```

Comment:
`trim_trailing_whitespace = false`
This should be set for md files in my opinion. Markdown has special meaning when there are 2 spaces at the end of the line.